### PR TITLE
Modal feature

### PR DESCRIPTION
**Pull request by marcusbarnesdeveloper**

### CODE REVIEW

**Comment by marcusbarnesdeveloper** [comment]
@geophray could pr please

File: client/singleItemPage.jsx [hunk 1]
```diff
@@ -2,20 +2,77 @@ import React from 'react';
...
+        sY: Math.abs(e.clientY)
+      });
+      console.log('hello');
   }
 
+onLeaveMain(e){
+  this.setState({
+    hovering: false
+  });
+}
```
Comment (geophray) [inline client/singleItemPage.jsx:58]: Fix indentation.

File: public/style.css [hunk 2]
```diff
@@ -36,9 +37,38 @@ body{
...
+  position: relative;
+}
+
+#mags{
+  background-image:url('https://images-na.ssl-images-amazon.com/images/G/01/apparel/rcxgs/tile._CB483');
+  cursor: pointer;
+  width: 227px;
+  height: 110px;
+  position: absolute;
 }
```
Comment (geophray) [inline public/style.css:74]: Add new line

File: server/app.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    }
+  });
+});
+
+
+
+
+
+
+module.exports = app;
```
Comment (geophray) [inline server/app.js:45]: Add new line

**Pull request merged**